.coveragerc

```diff
@@ -9,6 +9,11 @@ source =
 skip_covered = true
 skip_empty = true
 show_missing = true
+exclude_also =
+    if TYPE_CHECKING
+    assert False
+    : \.\.\.(\s*#.*)?$
```

Comment:
@bdraco Are these tripping coverage.py and not MyPy? Could you add code comments to justify each exclusion?